src/Features/CSharpTest/AddUsing/AddUsingTests.cs

```diff
@@ -6527,5 +6527,53 @@ class Class
                 index: 0,
                 parameters: new TestParameters(options: Option(FormattingOptions2.NewLine, configuredNewLine), testHost: testHost));
         }
+
+        [Theory]
+        [CombinatorialData]
+        [WorkItem("http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/867496")]
```

Comment:
Unexpected work item

Reply:
This was moved from another test class, and I preserved the work item. Should I update it based on the current issues it involves?

Reply:
Discussed offline. Don't move. Keep existing tests and update their expected behavior. 

You can def add new tests elsewhere as well (including with copies of the test state).